biofefi/machine_learning/get_models.py

```diff
@@ -17,33 +20,70 @@
     (ModelNames.XGBoost, ProblemTypes.Regression): XGBRegressor,
     (ModelNames.SVM, ProblemTypes.Classification): SVC,
     (ModelNames.SVM, ProblemTypes.Regression): SVR,
+    (
+        ModelNames.BayesianRegularisedNeuralNets,
+        ProblemTypes.Classification,
+    ): BayesianRegularisedNeuralNets,
+    (
+        ModelNames.BayesianRegularisedNeuralNets,
+        ProblemTypes.Regression,
+    ): BayesianRegularisedNeuralNets,
 }
 
 
 def get_models(
     model_types: Dict[str, Dict], problem_type: str, logger: object = None
 ) -> List:
+    """
+    Initialize and return a dictionary of models based on
+    specified types and problem domain.
+
+    Parameters
+    ----------
+    model_types : Dict[str, Dict]
+        Dictionary where keys are model names and values are
+        dictionaries containing model parameters and
+        usage flags.
+
+    problem_type : str
+        The type of problem ('classification' or 'regression').
+
+    logger : object, optional
+        Logger object for logging information.
+
+    Returns
+    -------
+    models : Dict[str, object]
+        Dictionary of initialized model instances.
+    """
```

Comment:
Can you change the style of the docstring here to match the rest of the app. Use the google style please

Reply:
Sure Dan. I'll change it.